### PR TITLE
Improvement/home page

### PR DESCRIPTION
**Pull request by Lucasha11**
Fixed Spacing in Header
Screenshot 2024-12-17 at 6 07 39 PM

### CODE REVIEW

File: content/pages/our-impact.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+blocks:
+  - logo:
+      src: /inreach-logos/InReach_Logo_Color_RGB.svg
+      alt: InReach Logo
+    navBGColor: Green
+    navLinks:
+      - linkName: About
+        linkUrls:
+          - sectionUrl: Mission
+            pathURL: /
```
Comment (ShawnAviles) [inline content/pages/our-impact.mdx]: these links still don't go to any actual page

**Comment by ShawnAviles** [comment]
@Lucasha11 What spacing is this fixing? No code was changed and you only deleted three sections from the navbar?

**Comment by Lucasha11** [comment]
There were 2 different codes for the header, one pushed the contact tab too close to the Donate button.

File: content/pages/our-impact.mdx (hunk 1 above)
Comment (ShawnAviles) [reply content/pages/our-impact.mdx]: nvm, i see that it is handled in #107

**Review by ShawnAviles — DISMISSED** [review]
Makes sense. Looks good

**Review by j-ferber — APPROVED** [review]
Good to merge

**Pull request merged**